packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -71,9 +74,14 @@ Request.prototype.query = function (queryParams) {
 
 Request.prototype.send = function (postParams) {
 	if (typeof postParams === 'undefined') {
-		return merge({}, this._postParams);
+		return merge({}, this._postParams || {});
+	}
+	if (postParams !== null) {
```

Comment:
I always get a little skittish when `undefined` and `null` are treated differently; is there some way we can consolidate them?